Formula/libxml2.rb

```diff
@@ -41,7 +41,7 @@ class Libxml2 < Formula
 
   keg_only :provided_by_osx
 
-  depends_on :python => :optional
+  depends_on :python if MacOS.version <= :snow_leopard
```

Comment:
Aye, but presumably `python` as `python2` would be hidden away as `keg_only`, so unless you remember to adjust your `$PATH` everything that needs the older `python` would be linking against `/usr/bin/python` and its questionable linkage, no?